app/src/main/java/org/kinecosystem/kinit/server/OfferService.kt

```diff
@@ -120,7 +120,7 @@ class OfferService(context: Context, private val offersApi: OffersApi, val userR
                     val transactionId = wallet.payForOrder(offer.address!!, offer.price!!, bookOfferResponse?.orderId!!)
 
                     wallet.updateBalanceSync()
-                    wallet.logSpendTransactionCompleted(offer.price, bookOfferResponse.orderId)
+                    wallet.logSpendTransactionCompleted(offer.price, transactionId.id())
```

Comment:
cool - can we also change the **name** of the parameter in logP2pTransactionCompleted, logSpendTransactionCompleted and logEarnTransactionCompleted to be "transactionId" instead of orderId?

Reply:
done